packages/medusa/src/api-v2/admin/inventory-items/route.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../types/routing"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+
+import { AdminPostInventoryItemsReq } from "./validators"
+import { ManyToManyInventoryFeatureFlag } from "@medusajs/utils"
+import { MedusaError } from "@medusajs/utils"
+import { Modules } from "../../../../../modules-sdk/dist"
+import { createInventoryItemsWorkflow } from "@medusajs/core-flows"
+import { defaultAdminProductsVariantFields } from "../products/query-config"
+
+// List inventory-items
+export const GET = async (
+  req: AuthenticatedMedusaRequest,
+  res: MedusaResponse
+) => {
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  const query = remoteQueryObjectFromString({
+    entryPoint: "inventory_items",
+    variables: {
+      filters: req.filterableFields,
+      order: req.listConfig.order,
+      skip: req.listConfig.skip,
+      take: req.listConfig.take,
+    },
+    fields: [...(req.listConfig.select as string[])],
+  })
+
+  const { rows: items, metadata } = await remoteQuery({
+    ...query,
+  })
+
+  const inventory_items = items.map((i) => {
+    const [stocked_quantity, reserved_quantity] = i.location_levels.reduce(
+      ([stocked, reserved], level) => [
+        stocked + level.stocked_quantity ?? 0,
+        reserved + level.reserved_quantity ?? 0,
+      ],
+      [0, 0]
+    )
```

Comment:
thought: Wondering if this should be computed in the module. Would that be possible?

Reply:
Lol, I guess we have the same thought. Saw this in your follow-up PR 😄

Reply:
yes, I'll add this in the list pr and remote the endpoint from the route-file 😄